dynamic/provider_test.go

```diff
@@ -264,7 +264,17 @@ func TestCheckConfig(t *testing.T) {
 		}),
 	}, expect(autogold.Expect(`{
   "inputs": {
-    "endpoint": 123.456
+    "endpoint": "123.456"
+  }
+}`))))
+
+	t.Run("nested-config", assertGRPCCall(s.CheckConfig, &pulumirpc.CheckRequest{
```

Comment:
Reading this test standalone I can't figure out what it's testing and asserting. Perhaps a better name in TestCheckConfig or a quick comment.

Reply:
I'll add a comment.